src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,21 @@
 package core.basesyntax;
 
+import static java.lang.System.out;
+
 public class Application {
+
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[NUMBER_OF_BALLS];
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = lottery.getRandomBall();
+        }
+
+        for (Ball ball : balls) {
+            out.println(ball);
+        }
```

Comment:
we can remove the array and leave just 1 loop where we print the result of `getRandomBall()` right away, but I guess this approach is Ok too :)